test/320_claim_3_test.sh

```diff
@@ -2,6 +2,8 @@
 
 . ./config.sh
 
+UNIVERSE=10.32.0.0/12
```

Comment:
Unless there is a very good reason to do otherwise, the universe for testing should be `10.2.0.0/16` or `10.2.x.0/24`. That is what we use everywhere else.